plugins/module_utils/foreman_helper.py

```diff
@@ -104,6 +104,21 @@ def _patch_location_api(self):
         _location_update_params_location = next(x for x in _location_update['params'] if x['name'] == 'location')
         _location_update_params_location['params'].append(_location_organizations_parameter)
 
+    def _patch_content_uploads_update_api(self):
```

Comment:
What happend to alphabetical? Also should this not be in the Katello Module?

Reply:
There is no KatelloNotEntityApypieModule (yet), so that's why it's here, and yeah I might have broken my own rules.

Reply:
I still think, we can make the difference between a Foreman* and a Katello* class a mixin.
ATM it is providing the api_patching and adding organization to argument_spec.

Reply:
you did, I used. great.